src/cript/nodes/primary_nodes/inventory.py

```diff
@@ -24,6 +24,32 @@ class Inventory(PrimaryBaseNode):
     | material  | list[[Material](./material.md)] |                     | material that you like to group together |
 
 
+    ## JSON Representation
+    ```json
+    {
+       "name":"my inventory name",
+       "node":["Inventory"],
+       "uid":"_:90f45778-b7c9-4b77-8b83-a6ea9671a937",
+       "uuid":"90f45778-b7c9-4b77-8b83-a6ea9671a937",
+       "material":[
+          {
+             "node":["Material"],
```

Comment:
That is actually not how it would come out if you called `.json`.
The material has to be condensed into UUID to be valid under the schema.

Reply:
I was thinking of not putting the UUID for the user for now because it might confuse them, but we could have that JSON and write a small section on condensing the node to its UUID and link to it in all places that it is condensed.

Reply:
yeah, ok that is fair.